forte/data/ontology/core.py

```diff
@@ -635,5 +637,203 @@ def index_key(self) -> int:
         return self.tid
 
 
+class Grid:
+    """
+    Regular grid with a grid configuration dependent on the image size.
+    It is a data structure used to retrieve grid-related objects such as grid
+    cells from the image. Grid itself doesn't store any data.
```

Comment:
I think the core functionality of a `Grid` is used to index the image area so that we can easily compute the overlap of any two areas. Let's don't forget this. 

Btw, is there any use case you find that we need to retrieve the image inside the grid cell?

Reply:
I think bounding box prediction is usually based on only image on inside a grid cell (anchor box) in YOLO algorithm.

Reply:
1. Is it a convention or just a choice of an algorithm?
2. you will need to explain these things to the users. before that, can you make sure you explain this to the team so people understand what you are doing?